en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,35 @@
+# Sbus Driver for linux（drivers/linux_sbus）
+This driver allows the autopilot to fetch data from the futaba sbus receiver via serial port, up to 16 channels.  
```

Comment:
Does this driver only supports Futaba? What about FrSky? This should be state here.